lib/src/render.dart

```diff
@@ -27,5 +27,23 @@ String render(String path,
       : new Stylesheet.parseScss(contents, url: url, color: color);
   var cssTree =
       evaluate(sassTree, color: color, packageResolver: packageResolver);
-  return toCss(cssTree, useSpaces: useSpaces, indentWidth: indentWidth);
+  return toCss(cssTree,
+      useSpaces: useSpaces, indentWidth: indentWidth, linefeed: linefeed);
+}
+
+LineFeed parseLineFeed(String str) {
```

Comment:
It seems like this should be a private method in node.dart.

Also, please document it.